templates/cpp-template-default/Classes/HelloWorldScene.cpp

```diff
@@ -30,9 +37,18 @@ bool HelloWorld::init()
                                            "CloseNormal.png",
                                            "CloseSelected.png",
                                            CC_CALLBACK_1(HelloWorld::menuCloseCallback, this));
-    
-    closeItem->setPosition(Vec2(origin.x + visibleSize.width - closeItem->getContentSize().width/2 ,
-                                origin.y + closeItem->getContentSize().height/2));
+    auto w = closeItem->getContentSize().width;
+    auto h = closeItem->getContentSize().height;
+    if (w <= 0 || h <= 0)
```

Comment:
No, there is not problem.

@TheCodez good catch, i think it should check `closeItem` too.